documentation/modules/rhv-prerequisites.adoc

```diff
@@ -9,7 +9,6 @@
 The following prerequisites apply to {rhv-full} migrations:
 
 * You must use a xref:compatibility-guidelines_{context}[compatible version] of {rhv-full}.
-* You must have the CA certificate of the {manager}.
+* You must have the CA certificate of the {manager} or the Apache CA certificate of the {manager} if the {manager} Apache CA certificate was replaced by a third-party CA certificate.
```

Comment:
@RichardHoch 
For consistency throughout the doc:
"* You must have the {manager} CA certificate or the {manager} Apache CA certificate if the {manager} CA certificate was replaced by a third-party certificate on the Apache server."